jest.config.js

```diff
@@ -13,13 +16,13 @@ module.exports = {
   coverageReporters: ['lcov'],
   reporters: ['jest-junit', 'default'],
   transform: {
-    '^.+\\.(js|jsx)$': path.join(__dirname, './test/jest/jest-transformer.js'),
+    '^.+\\.(js|jsx|ts|tsx)$': path.join(__dirname, './test/jest/jest-transformer.js'),
```

Comment:
do we have typescript?

Reply:
We do!
https://github.com/folio-org/stripes-webpack/search?q=typescript

Reply:
does someone use it? I don't see reason to support ts when our modules are written in js

Reply:
My understanding is that using the TypeScript is enabled for any module to be able to use it if needed.
As stripes-acq-components is a sub-library of components for anyone to use, I guess we should support that concept.

Also, I'm going to use TS )